cli/src/commands/extensions/mod.rs

```diff
@@ -114,6 +125,35 @@ pub async fn handle_run_extension(
     Ok(CommandValue::Code(ExitCode::Ok))
 }
 
+/// Handle the `extension run <PATH>` command path.
+///
+/// Run the extension located at the given path.
+pub async fn handle_run_local_extension(
+    api: BoxFuture<'static, Result<PhylumApi>>,
+    matches: &ArgMatches,
+) -> CommandResult {
+    let path = matches.value_of("PATH").unwrap();
+    let options = matches.get_many("OPTIONS").map(|options| options.cloned().collect());
+
+    let extension_path = PathBuf::from(path);
+    let extension = Extension::try_from(extension_path)?;
+
+    println!("{:?}", extension);
+
+    // Attempt to construct a `PermissionsOptions` from the `Permissions`
+    // object in order to validate the permissions.
+    let permissions = extension.permissions();
+    let _ = PermissionsOptions::try_from(permissions.borrow())?;
```

Comment:
This seems like a really hacky way to "validate" permissions.

Reply:
I copied it over verbatim from `handle_install_extension`. What could be a better way?

Reply:
There's a `From<&Permissions> for PermissionsOptions` impl, so isn't this basically just a no-op that will **always** succeed anyway? I think this code might be pointless.

Reply:
Indeed.